src/draco/compression/attributes/normal_compression_utils.h

```diff
@@ -318,7 +318,7 @@ class OctahedronToolBox {
 
     // Remaining coordinate can be computed by projecting the (y, z) values onto
     // the surface of the octahedron.
-    const float x = 1.f - abs(y) - abs(z);
```

Comment:
in gcc/clang, C function `abs` have prototype `int abs(int)` which will implicitly cast float to int and return a integer.